README.md

```diff
@@ -8,10 +8,17 @@ Please read the [contribution guidelines](CONTRIBUTING.md).
 
 ## Setting up Development Environment
 
-Floworky uses [`node-foreman`](https://github.com/strongloop/node-foreman) to manage process startup.  This requires the use of a `.env` file to specify the environment variables required by floworky (note that this is an example that includes values that will need to be set for your own environment):
+Floworky uses [`node-foreman`](https://github.com/strongloop/node-foreman) to  
```

Comment:
Could you reconcile what you're saying against the following fromm the creator of Markdown?
> The overriding design goal for Markdown’s formatting syntax is to make it as readable as possible. The idea is that a Markdown-formatted document should be publishable as-is, as plain text

I can think of many instances when the README.md and <other>.md files are consumed unrendered, as plain text. In fact, I —as an anecdote— primarily consume the README.md via `cat` or vim in Terminal and via my text editor while working. Why wouldn't one want to make the content as accessible as they could —in this case, by improving readability by observing best practices regarding line length— to even users consuming it as plain text.